cnames_active.js

```diff
@@ -3117,6 +3117,7 @@ var cnames_active = {
   "ushio": "ushio.netlify.com",
   "usv": "usvformat.github.io",
   "utc": "pythonscratcher.github.io/utc",
+  "utilities": "blob2763.github.io/utils"
```

Comment:
```suggestion
  "utilities": "blob2763.github.io/utils",
```
As the CI error notes, you are missing a comma at the end of your entry